spec/influxdb/cases/query_with_params_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "spec_helper"
+
+describe InfluxDB::Client do
+  let(:subject) do
+    described_class.new "database", {
+      host: "influxdb.test",
+      port: 9999,
+      username: "username",
+      password: "password",
+      time_precision: "s"
+    }.merge(args)
+  end
+
+  let(:args) { {} }
+
+  describe "#query with parameters" do
+    let(:query) { { "SELECT value FROM requests_per_minute WHERE time > :start:" => { start: 1_437_019_900 } } }
+    let(:response) do
+      { "results" => [{ "series" => [{ "name" => "requests_per_minute",
+                                       "columns" => %w( time value ) }] }] }
+    end
+
+    before do
+      stub_request(:get, "http://influxdb.test:9999/query")
+        .with(query: { db: "database", precision: "s", u: "username", p: "password", q: "SELECT value FROM requests_per_minute WHERE time > 1437019900" })
+        .to_return(body: JSON.generate(response), status: 200)
+    end
+
+    it "should handle responses with no values" do
+      # Some requests (such as trying to retrieve values from the future)
+      # return a result with no "values" key set.
+      expected_result = [{ "name" => "requests_per_minute", "tags" => nil, "values" => [] }]
+      expect(subject.query(query)).to eq(expected_result)
+    end
+  end
```

Comment:
I've added the above tests, and after some added code, they pass (raising an argument error), but now rubocop is at it again.